dev/tasks/homebrew-formulae/autobrew/apache-arrow-static.rb

```diff
@@ -0,0 +1,102 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Cloned from https://github.com/autobrew/homebrew-core/blob/high-sierra/Formula/apache-arrow.rb
+# Upstream any relevant changes (dependencies, build args ...)
+
+class ApacheArrowStatic < Formula
```

Comment:
Can we use different name such as `ApacheArrowHighSierra` or something?
It seems that both of `apche-arrow.rb` and `apache-arrow-static.rb` build static libraries.

Reply:
I also thought the name was not ideal but this is the formula name in autobrew: https://github.com/autobrew/homebrew-cran/blob/master/Formula/apache-arrow-static.rb So I think we should keep it aligned with the upstream name.

Reply:
Oh. Then, could you update the URL in "Cloned from https://..." in the above comment?